### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by marianaWest**
Hello! I did some work on the layout, it's not great yet but a little better. Baby steps.

### CODE REVIEW

File: views/partials/header.ejs [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Red Balloon</title>
+    <title >Red Balloon</title>
```
Comment (emtek995) [inline views/partials/header.ejs:7]: Maybe erase the space.

File: package.json [hunk 2]
```diff
@@ -18,11 +18,21 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marianaWest/birthdayApp.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/marianaWest/birthdayApp/issues"
+  },
+  "homepage": "https://github.com/marianaWest/birthdayApp#readme"
```
Comment (emtek995) [inline package.json:37]: Maybe remove this or change to 40plusbaddies repo

**Review by emtek995 — CHANGES_REQUESTED** [review]
Just a couple things to fixup.
Great work!

File: views/partials/header.ejs (hunk 1 above)
Comment (marianaWest) [reply views/partials/header.ejs]: I think I did it, let me know if it doesn't work.

**Review by marianaWest — COMMENTED** [review]
Hey @emtek995 thanks for the review. I think it's all good now, but my commit also has a newer version of mongoose -- should we keep the older version?

**Comment by borojetski** [comment]
Looks good!

**Pull request merged**